status_and_problems.tex

```diff
@@ -66,22 +61,27 @@ \subsubsection{Build Tools} \label{sec:statusTools}
 
 The Jenkins scripts also are an important part of the tooling, facilitating a large quantity of actions, like, test a ticket branch, automate releases, or provide periodic build (weekly/daily).
 
-The tools picture is completed by \textit{codekit}, that permits to interact with multiple GitHub packages at the same time.
-Just as an example, it is used to create a tag on all the GitHub packages composing a defined product.
+The tools picture is completed by \textit{codekit}, which permits interaction with multiple GitHub packages.
+Just as an example, it is used to create the same tag on all the GitHub packages composing a defined product automatically, instead of creating it manually.
 
 \textit{TO BE CLARIFIED: which are java build tools?}
 
 
 \subsection{Open Problems} \label{sec:openProblems}
 
-\subsubsection{SW Product Identification} \label{sec:problemId}
+Follows the list of problems derived from the actual development approach. 
+
+Until these problems are not resolved, the only release process suitable for DM is to release all codebase at the same time.
```

Comment:
Perhaps - While these problems are unresolved the only release process suitable for DM is to release the entire codebase each time.